.github/workflows/reports.yml

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: 'Reports'
+on:
+  workflow_run:
+    workflows: [ 'Test' ]
+    types:
+      - completed
+
+jobs:
+    report:
+        runs-on: ubuntu-latest
```

Comment:
It is a small thing... preference for 2 spaces as the indent instead of 4 in YML files. We have that specified in the `.editorconfig` file at the root of the project. Do you have the editorconfig extension for VS Code? It will set VS Code to use 2 spaces automatically.

Reply:
I will review this. I think I do but not sure why it didn't do it

Reply:
I found that there are two `.editorconfig`  files, one in the root and one in Meadowlark-js. I think this is needed depending if you open VS Code from the root or from the Meadowlark-js folder. Those two files had different setup so I matched the settings.